client/src/pages/AddProject.js

```diff
@@ -101,13 +103,11 @@ class AddProject extends Component {
         const validation = this.validators.validate(this.state);
```

Comment:
While you are at it, can you check the validation/bugs of when a user does not set/change fundingGoal?

Reply:
I shall look into it